tsconfig.json

```diff
@@ -8,6 +8,9 @@
     "jsx": "react",
     "esModuleInterop": true
   },
+  "include": [
+    "./scripts/setupTests.ts"
```

Comment:
@marksy do you remember why this was needed? It broke other typescript features, e.g. you can no longer command click/go to definition. Removing this fixes the issue.

Reply:
For jest set up iirc